res/database/Default/user/scr0llbaer.xml

```diff
@@ -0,0 +1,1586 @@
+					<en>Bali|Laos|Cambodia|Myanmar|Bhutan|Nepal|Bangladesh|Sri Lanka|Tibet|Vietnam|Macau|Kalmykia</en>
+				</bali>
+			</variables>
+
+			<jobs>
+				<job index="0" function="1" required="1" />
+				<job index="1" function="4" required="1" />
+				<job index="2" function="128" required="1" />
+			</jobs>
+
+			<data flags="4" production_limit="412" production_broadcast_limit="2" production_licence_flags="36" />
+			<genres maingenre="6" subgenres="300" />
+			<blocks value="2" />
+			<price min="3000" max="6000" />
+			<potential min="20" max="70" />
+			<outcome min="20" max="70" />
+			<review min="30" max="80" />
+			<speed min="20" max="50" />
+			<programme_data_modifiers> <!-- Idea is that while the content itself may be kind of ageless and could reach Discovery Channel-like quality, the episodes are nevertheless meant to be discarded quickly (only 2 broadcasts) as there are too many gods to cover, and as to not spam one's archive. -->
+				<modifier name="topicality::age" value="0.1" />
+				<modifier name="topicality::refresh" value="0.5" />
+			</programme_data_modifiers>
+		</scripttemplate>
+
+		<scripttemplate product="1" licence_type="1" guid="352a2fbc-7e7b-462b-a217-34c300043d19" creator="9551" comment="inspired from https://en.wikipedia.org/wiki/Maniac_Mansion">
+			<title>
+				<de>%title%</de>
+				<en>%title%</en>
+			</title>
+			<description>
+				<de>Dylans Freundin Sara wurde entführt vom verrückten Professor Dr. Ed! Er und seine Freunde wollen sie aus dessen Villa befreien. Dort gehen noch viel mehr merkwürdige Dinge vor sich! Und wo ist Benzin für die Kettensäge, wenn man's mal braucht.</de>
+				<en>Dylan's girlfriend Sara has been kidnapped by the mad professor Dr. Ed! He and his friends want to free her from his mansion. There are a lot more strange things going on there! And where is gasoline for the chainsaw when you need it.</en>
+			</description>
+
+			<variables>
+				<title>
+					<de>Der %adj% %house-m%|Die %adj% %house-f%|Das %adj% %house-n%</de>
+					<en>The %adj% %house-m%|The %adj% %house-f%|The %adj% %house-n%</en>
+				</title>
+				<adj>
+					<de>Wahnwitzige|Verrückte|Irrwitzige|Durchgeknallte|Geisteskranke|Irrsinnige|Aberwitzige|Manische|Wahnsinnige|Absurde|Groteske|Bizarre|Abgedrehte</de>
+					<en>Lunatic|Crazy|Mad|Frenzied|Frantic|Insane|Ludicrous|Manic|Maniacal|Absurd|Grotesque|Bizarre|Wacky</en>
+				</adj>
+				<house-m>
+					<de>Landsitz</de>
+					<en>Country Estate</en>
+				</house-m>
+				<house-f>
+					<de>Villa|Residenz</de>
+					<en>Villa|Residence</en>
+				</house-f>
+				<house-n>
+					<de>Herrenhaus|Gutshaus|Anwesen|Landhaus|Tollhaus</de>
+					<en>Manor|Farmhouse|Estate|Country House|Madhouse</en>
+				</house-n>
+			</variables>
+
+			<jobs>
+				<job index="0" function="1" required="1" />
+				<job index="1" function="2" required="1" gender="1" role_guid="2ebfca2c-0ac3-450d-a4cb-e3f7e90904f2" />
```

Comment:
when the version with that feature is released. Adding todo.

Reply:
I think it would be nice to use the new features already with the coming release. Maybe we can put together a pre-release, so you can test already?

Reply:
Would need to know the OS you are using (Windows, Linux, Mac).

You can also join the discord server if you want so I can easily hand out stuff (and for live help by me :D). Github/Forum/Mail is also fine.

Reply:
Linux. Arch (btw)

I don't see a discord link anywhere. I have an account but I don't use it much (too bloated).

I have git-cloned the whole project and *could* try to get yet another dev environment running but with this project I would like to specialize on the data, not on the code. `:)`

Reply:
Once the live-programme fix is merged, it would be quite nice if you had a snapshot release. The additional show formats for different topics are "much needed", so it would be great to include them (after you had the chance for some fine-tuning).